### DISCUSSION TITLE
How is Min/Max Calculated?

### DISCUSSION THREAD

**Discussion by gary-hammer** [bellrichm/weewx-jas#86] [Q&A]
How is the Min/Max calculated?
I ask because today, we had a fairly strong front go through.
Both the seasons skin for weewx and the Davis Weatherlink site show much higher wind and rain rates.

Max Wind | 25 seasons and Davis
Wind Speed | 0 mph 20:25 | 11 mph 12:05 jas 0.3.0

Rain Rate | 5.65 | in/h seasons and Davis
Rain Rate | 0 in/h 23:15 | 1.4 in/h 07:10 jas 0.3.0

**Comment by bellrichm** [comment]
It looks like seasons uses wind and jas uses windSpeed. Starting in 4.9.2, jas might be able to use wind…
For reference, http://weewx.com/docs/customizing.htm#Wind

rainRate should match… what does the chart look like. If you attach the data/last24hours.js, I can look.

**Comment by gary-hammer** [comment]
As the data comes from the WLL for all the output shouldn't all the values be equal between the skins and the Davis website?

Attaching the requested file. But, I'm afraid the data is gone at this point.
[last24.zip](https://github.com/bellrichm/weewx-jas/files/9997986/last24.zip)

**Comment by bellrichm** [comment]
[Commit](https://github.com/bellrichm/weewx-jas/commit/1cca8d2d235a721c504afd2f54566b9597bf0a8a) enables the use of the wind tag. This should match the season skin wind max value. It requires WeeWX 4.9.1, so I haven’t included it in the install yet.
I’m still puzzled by the rain rate mismatch. I’ll keep digging.
